hyperspy/io_plugins/tiff.py

```diff
@@ -20,10 +20,18 @@
 import warnings
 
 import traits.api as t
-with warnings.catch_warnings():
-    warnings.simplefilter("ignore")
-    from hyperspy.misc.io.tifffile import imsave, TiffFile
 from hyperspy.misc import rgb_tools
+try:
+    from skimage.external.tifffile import imsave, TiffFile
+    raise ImportError
```

Comment:
As you may have guessed, that was a debugging import. It should be fixed now, thanks!